### PR TITLE
Fix personal note UI

### LINKED ISSUE #2621: Motion detail view: Set Star-Icon generates undesired text in personal comment
**Description:** If a user activates the star icon in a motion, the text 'undefined' appears the personal comments field and won't disappear until it is manually deleted. 

**image without active star**
Screenshot_20230811_135955

**image with active star**
Screenshot_20230811_140009

**Reproduction:**
1. open a meeting
2. open motions > create a motion
3. open the motion > activate star-icon 

**What should happen:** 
After activating the star-icon no text should appear in the personal comment field.

### PR DESCRIPTION
**Pull request by luisa-beerboom**
Closes #2621

### CODE REVIEW

**Review by bastianjoel — APPROVED** [review]

**Review by Elblinator — APPROVED** [review]
function

File: client/tests/integration/meeting/motion/history.spec.ts
Comment (bastianjoel) [inline client/tests/integration/meeting/motion/history.spec.ts:1]: This should not be included

**Review by bastianjoel — CHANGES_REQUESTED** [review]

File: client/tests/integration/meeting/motion/history.spec.ts
Comment (luisa-beerboom) [reply client/tests/integration/meeting/motion/history.spec.ts:1]: Why not?

Comment (bastianjoel) [reply client/tests/integration/meeting/motion/history.spec.ts:1]: It is unrelated to the issue. We could make another PR where we do a full cleanup of those. Maybe we could also add the test directory to the cleanup command.

**Review by bastianjoel — APPROVED** [review]

**Pull request merged**